app/Resources/views/Form/fields.html.twig

```diff
@@ -0,0 +1,17 @@
+{#
+   Each field type is rendered by a template fragment, which is determined
+   by the value of your getName() method and the suffix _widget.
+
+   See http://symfony.com/doc/current/cookbook/form/create_custom_field_type.html#creating-a-template-for-the-field
+#}
+
+{% block app_datetimepicker_widget %}
+    {% spaceless %}
+        <div class='input-group date' id="datetimepicker">
```

Comment:
@yceruto With css class you could reuse it for more than one field. With ID it's impossible if we want to keepvalid HTML structure.